### ISSUE TITLE
Add job queue for managing scraping

### ISSUE THREAD

**Issue by Mr0grog** [edgi-govdata-archiving/web-monitoring-db#3]
We’ll want this for two things:

1. Better control of how/when to scrape (currently using Heroku Scheduler to run a rake task; not great)
2. Ability to divide scraping work up over several processes or machines working in parallel so it goes much faster.

**Comment by Mr0grog** [comment]
Looks like the Heroku scheduler is cutting off scraping jobs, so support for (1) needs to be done sooner rather than later. Manually updating the DB for now :(

**Comment by Mr0grog** [comment]
No longer relevant—the DB now simply receives content from other tools whenever they send it. Closing this for now unless we want to come back to the DB being a coordinator later.
